tutorial/how_to_build.md

```diff
@@ -0,0 +1,17 @@
+If you are building this project from source, you can try the following
+
+```
+git clone https://github.com/spark-redshift-community/spark-redshift.git
+```
+
+```
+cd spark-redshift
+```
+
+```
+sbt -v compile
+```
+
+```
+sbt -v package
```

Comment:
I generally use the sbt binary in build/ (it's the project version 0.13.13 which might differ from the system version):
```
./build/sbt package
```

Also to test (unit and integration):
```
./build/sbt test
./build/sbt it:test
```

Note that before it:test they need to set up all the evnironment variables to connect to Redshift (see https://github.com/spark-redshift-community/spark-redshift/blob/master/src/it/scala/io/github/spark_redshift_community/spark/redshift/IntegrationSuiteBase.scala#L54).

Reply:
Good points, will incorporate the changes.